docs/source/exposing.md

```diff
@@ -74,6 +74,36 @@ For the `LoadBalancer` type, Scylla Operator generates a LoadBalancer Service th
 On platforms with support for external load balancers, this Service provisions one. 
 The accessibility of this load balancer's address depends on the platform and any customizations made; in some cases it may be reachable from the internal network or public Internet.
 
+Customizations are usually managed via Service annotations, key-values provided in `annotations` field are merged into each Service object.
```

Comment:
Although the options imply it, it might be worth to add a note that they should configure it in passthrough mode and not redirect. (For other platforms and better background.)